### DISCUSSION TITLE
Grosso problema a configurare ZigBee mqtt

### DISCUSSION THREAD

**Discussion by alenyc81** [indomus/forum#2744] [HUB: Home Assistant]
Ciao a tutti, qualcuno ha modo di aiutarmi a fare funzionare zigbeemqtt su raspberry?
Ho già mqtt broker, ho seguito le varie guide per zigbeemqtt ma non si avvia nemmeno. Probabilmente c'è qualche problema di configurazione...

Questa è la schermata di configurazione

[Uploading Screenshot_20231203_170138_Home Assistant.jpg…]()

Questa invece la configurazione di mqtt broker

È da 2 ore che ci provo ma nulla

Grazie in anticipo ciao

**Comment by snakuzzo** [comment]
Impossibile aiutarti.
1. non hai postato alcuno screenshot
2. ci facciamo poco con gli screenshot, servono i log...e possibilmente non incollati come screenshot ma come testo utilizzando il markdown.

**Comment by alenyc81** [comment]
Niente, alla fine ,non so come, sono riuscito ad avvisarlo... Però non ho risolto il problema rispetto a zha... Speravo di avere dei parametri  per poter fare il ping dei sensori temperatura più frequentemente,invece niente

**Reply by Scialla** [reply]
non puoi forzare la comunicazione dei sensori a batteria, puoi farlo con quelli alimentati. Se ci pensi non avrebbe nemmeno senso farlo, la batteria durerebbe pochissimo aumentando la trasmissione dei dati.
I sensori poi si "risvegliano" e inviano il nuovo dato al variare dello stesso. Cher senso ha che ti rimandano lo stesso valore continuamente?

**Comment by alenyc81** [comment]
Il problema è questo:

.. Spesso vedo come ultimo aggiornamento stato anche mezz'ora... Avendo collegato questi sensori all'automazione riscaldamento in casa, spesso e volentieri mi trovo con il riscaldamento spento perché ,le condizioni di temp minima per fer partire il riscaldamento ci sono, ma i sensori non vengono interrogati con frequenza...

**Reply by Scialla** [reply]
Ribadisco quanto già scritto di là (evitiamo un doppio post) e quanto già detto qua: i sensori a batteria NON vengono interrogati da nessuno, sono i sensori stessi a trasmettere, quando questo varia, il valore rilevato. NON si possono interrogare, e questo perché vanno in deep sleep e si risvegliano da soli quando il valore cambia
